addons/mail/models/discuss/mail_guest.py

```diff
@@ -136,5 +137,7 @@ def _guest_format(self, fields=None):
                 data['name'] = guest.name
             if 'im_status' in fields:
                 data['im_status'] = guest.im_status
+            if "write_date" in fields:
+                data["write_date"] = guest.write_date.strftime(DEFAULT_SERVER_DATETIME_FORMAT)
```

Comment:
Can you try with `to_string()` on `write_date`? I think it gives the same result in a much simpler manner.

Reply:
It's a bit messy because in partner format and guest format we have a variable called `fields` that override the `odoo.fields` import :disappointed: 
I used `import odoo` and `odoo.fields.Datetime.to_string`. Not great I guess?

Reply:
import odoo is fine, that works for me.